ArchUnitNET/Fluent/Extensions/MonoCecilMemberExtensions.cs

```diff
@@ -158,18 +171,43 @@ public static bool IsBackingField(this FieldReference fieldReference)
 
         public static Visibility GetVisibility(this MethodDefinition methodDefinition)
         {
-            return methodDefinition.IsPublic ? Visibility.Public : Visibility.Private;
+            if (methodDefinition.IsPublic)
```

Comment:
maybe it will be better to put member visibility into definition?

Reply:
MethodDefinition is an extern class in Mono Cecil, so I can't implement it there.